### PR TITLE
added the http request automation documentation

### PR DESCRIPTION
**Pull request by PavelLinearB**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=oNBjGoF)
image

image

### CODE REVIEW

**Review by vim-zz — COMMENTED** [review]
1. It's better to call it Pro rather than Premium
2. The Beta icon is not self explanatory, maybe different one
3. How about adding in the action description an Admonition that exaplin that this is a Pro feature that will be charged in the future

File: docs/context-variables.md [hunk 1]
```diff
@@ -128,6 +134,45 @@ automations:
...
+  slack_message:
+    if:
+      - true
+    run:
+      - action: http-request@v1
+        args:
+          url: {{ slack.base }}{{ slack.channel }}
+          method: POST
+          headers: '{"Content-type": "application/json"}'
+          body: '{"text": "Hello, world!"}'
```
Comment (BenLloydPearson) [inline docs/context-variables.md:170]: Something that would help me generate examples is to see how to post the PR title and a link to the PR in the slack message. Can you modify this example to demonstrate that? I also think that would be a good starting point for users who are first learning how to use this.

File: docs/automation-actions.md [hunk 2]
```diff
@@ -179,6 +182,44 @@ automations:
       - action: close@v1
 ```
 
+#### `http-request` :fontawesome-solid-flask: :fontawesome-solid-crown: :fontawesome-brands-github: 
+
+!!! note "Pro feature, in open beta"
+	This feature is under development.
+	Once released, using this feature will be limited to pro subscriptions.
```
Comment (BenLloydPearson) [inline docs/automation-actions.md:189]: Will the entire feature be limited, or portions of it? If the latter is correct, we may want to indicate that people who adopt this won't suddenly lose access to everything unless they pay us at an unspecified point in the future.

That might make people feel more comfortable adopting  this feature now.

**Review by BenLloydPearson — CHANGES_REQUESTED** [review]

**Comment by BenLloydPearson** [comment]
Great work @PavelLinearB! I'll start work on some examples to support this.

**Comment by vim-zz** [comment]
@PavelLinearB also change the `Pro feature` to `Business tier feature`

**Pull request closed without merging**